contracts/test/TestUtilityBrandedToken.sol

```diff
@@ -45,7 +44,7 @@ contract TestUtilityBrandedToken is UtilityBrandedToken {
      * @param _organization Address of the Organization contract.
      */
     constructor(
-        EIP20Interface _token,
+        address _token,
```

Comment:
Same question as above Do we need to update this?
We can typecast EIP20Interface  with address and pass to UtilityToken.
What do you think?

Reply:
Same as above